docs/install.rst

```diff
@@ -1,5 +1,36 @@
 Installing JWST_REFFILES
 ========================
 
+Note that *jwst_reffiles*' main dependency, the `JWST calibration pipeline <https://github.com/spacetelescope/jwst>`_ requires Python 3.5 or above.
 
-Installation documentation goes here.
+Environment Creation
+--------------------
+
+It may be easiest to create an environment to contain *jwst_reffiles* by first creating an environment around the JWST calibration pipeline. See the installation instructions in the `JWST repo <https://github.com/spacetelescope/jwst>`_
+
+JWST_REFFILES installation
+--------------------------
+
+To download and install *jwst_reffiles*, clone from `GitHub <https://github.com/spacetelescope/jwst_reffiles>`_. Make sure you are in your desired environment, and then install using setup.py.
+
+1. git clone https://github.com/spacetelescope/jwst_reffiles.git
+2. cd jwst_reffiles
+3. python setup.py install
```

Comment:
maybe running setup.py takes care of this, but when I used the old method (pip install .), I had to add the location of jwst_reffiles to my PYTHON_PATH in my .bashrc file for it to work. If setup.py doesn't do this, may be worth mentioning you have to do this somewhere in the install instructions.